test/axios2.test.js

```diff
@@ -144,7 +192,9 @@ describe("Axios AuthHttpRequest class tests", function() {
                     exception = ex;
                 }
                 assertNotEqual(exception, undefined);
-                assertNotEqual(exception.response, undefined);
+                assertEqual(exception.status, 500);
+                assertNotEqual(exception.data, undefined);
+                assertEqual(exception.data.message, "test");
```

Comment:
The `exception` format still seems to be incorrect. We need to make it `exception.response.status`, `exception.response.data` in case of response being `500`. See https://stackoverflow.com/a/51768316/3867175